capsules/classifier_face_age_gender_openvino/backend.py

```diff
@@ -11,6 +11,20 @@
 from . import config
 
 
+def _get_bin(number: float):
+    bins = config.age_bins
+    if len(bins) is 0:
+        raise Exception('When binning, bin array must be larger than 0')
+
+    age_bin = next(iter(config.age_bins.keys()))
```

Comment:
No reason for this line here if bin size is guaranteed to be non-zero
```python

for min_age, category in config.age_bins.items():
    if number < min_age:
        return category

return category
```

Reply:
Also it's strange that the value that the bin is defined with is not in the bin. E.g. `12` will be put in the `13` bin

Reply:
alex and I squabbled a bit over this as well. How would you suggest it be written? We are trying to put the age into a bin. And if we find that the age is less than the current bin, we know that the previous bin is the correct one.

Reply:
use `<=` instead?

Reply:
Now that you've decided that my way is superior for now. Please approve the PR ;)